src/Microsoft.Framework.Configuration/ConfigurationBuilder.cs

```diff
@@ -27,11 +27,11 @@ public ConfigurationBuilder(string basePath, params IConfigurationSource[] sourc
             BasePath = basePath;
```

Comment:
Ah, I guess you started this before @kirthik's PR is in... Good luck merging! :smile: